java/java-hello-world/src/test/java/cloudcode/helloworld/SampleJettyApplicationTests.java

```diff
@@ -24,6 +24,5 @@ public void testHome() throws Exception {
 
 		ResponseEntity<String> responseEntity = this.testRestTemplate.getForEntity("/", String.class);
```

Comment:
You really should add more tests. Also I would expect this file name to match the file it's testing, ie HelloWorldControllerTests.java